src/main/java/uk/gov/companieshouse/search/api/mapper/ApiToResponseMapper.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.search.api.mapper;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.search.api.model.response.ResponseObject;
+
+@Component
+public class ApiToResponseMapper {
+
+    public ResponseEntity map(ResponseObject responseObject) {
+
+        switch(responseObject.getStatus()) {
+            case SEARCH_FOUND:
+                return ResponseEntity.status(HttpStatus.FOUND).body(responseObject.getData());
```

Comment:
We could use static imports to avoid having to provide qualified(?) constants like `HttpStatus.FOUND` (for brevity).

Reply:
Updated to use static imports.